hail/python/hail/docs/_static/rtd_modifications.css

```diff
@@ -130,7 +127,7 @@ div.nboutput div.output_area.stderr {
 
 .bolditalic {
   font-weight: bold;
-  font-italic: italic;
+  font-style: italic;
```

Comment:
too small for a separate PR I think. a mistake I noticed while making the present update.